lib/registries/addon/components/comment-card/component.ts

```diff
@@ -60,21 +60,20 @@ export default class CommentCard extends Component.extend({
         this.toast.success(this.i18n.t('registries.overview.comments.retract_report.success'));
     }),
     loadReplies: task(function *(this: CommentCard, more = false) {
-        let replies = yield this.comment.replies;
-
-        if (!more && replies) {
+        let replies;
+        if (!more) {
+            replies = yield this.comment.replies;
             this.set('replies', replies);
             return;
         }
 
         replies = yield this.comment.queryHasMany('replies', {
-            page: more ? this.incrementProperty('page') : this.set('page', 1),
+            page: this.incrementProperty('page'),
+            embed: ['user'],
         });
 
-        if (more && this.replies) {
+        if (replies) {
```

Comment:
`replies` will never be falsy, even if there are no replies. Is there there any reason not to use `queryHasMany` always?

```ts
loadReplies: task(function *(this: CommentCard, more=false) {
    const replies = yield this.comment.queryHasMany(...);
    if (more && this.replies) {
        this.replies.pushObjects(replies);
    } else {
        this.set('replies', replies);
    }
})
```

Reply:
This makes sense. Given the nature of replies/comments, I don't think we can embed the first page of replies on the comment. The logic (borrowed from contributors) was to retrieve the first page of replies as embeds on the comment.